Update all-contributors and move directory into tasks/ (#1062)
redwoodjs/redwood: CONTRIBUTORS.md

# Contributors Welcome!
## All Redwood Contributors 🎉
Redwood has a vibrant community we want to highlight as much as possible.

To view the current list of amazing contributors across Redwood projects, see the [Contributors section of README.md](https://github.com/redwoodjs/redwood/blob/main/README.md#contributors).

### Want to become a Contributor?

> Start with our [Contributing Doc](https://redwoodjs.com/docs/contributing). And then let us know how we can help get you up to speed by reaching out via the [Forums](https://community.redwoodjs.com) or [Discord chat](https://discord.gg/jjSYEQd).

Author: ```suggestion
First, read the [Contributing Doc](https://redwoodjs.com/docs/contributing). And then let us know how we can help you get started by reaching out via the [Forums](https://community.redwoodjs.com) or [Discord chat](https://discord.gg/jjSYEQd).
```
